packages/manager/modules/pci/src/projects/project/notebooks/Notebook.class.js

```diff
@@ -271,6 +272,17 @@ export default class Notebook {
     return this.spec.resources.ephemeralStorage;
   }
 
+  get workspaceFreeStorage() {
+    return (
+      this.status?.workspace?.storageFree ||
+      NOTEBOOK_FREE_WORKSPACE_STORAGE_DEFAULT
+    );
```

Comment:
**suggestion**: Could be helpful if `this.status?.workspace?.storageFree` can be `0`, `false`.
```suggestion
    return (
      this.status?.workspace?.storageFree ??
      NOTEBOOK_FREE_WORKSPACE_STORAGE_DEFAULT
    );
```

Reply:
That would be great, but when I tried it, the manager would not compile anymore with the following error: `SyntaxError: Assigning to rvalue `. 
I don't understand why, as I tried the same on a pure js project, and it worked. Do you have any idea why it would not compile ? 
That being said, `this.status.workspace.storageFree` should never be `0` or `false`, it should either exists and be a number above 0,  or not exists.
If you have an idea why it does not work, or how to solve it, let me know !

Reply:
If the value cannot be `0`. Then we can proceed with `||`.